crates/bevy_text/src/text.rs

```diff
@@ -160,6 +160,7 @@ impl From<TextAlignment> for glyph_brush_layout::HorizontalAlign {
 #[derive(Clone, Debug, Reflect, FromReflect)]
 pub struct TextStyle {
     pub font: Handle<Font>,
+    // The size of the font in pixels. Note that this value may surprise users who have a custom scale factor set.
```

Comment:
```suggestion
    /// The size of the font in pixels. 
    ///
    ///Note that this value may surprise users who have a custom scale factor set.
```

This needs to use doc comments, and splitting it into lines like this gives better hints / layout in IDEs.